package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "smart-array-filter",
-  "version": "4.1.1",
-  "main": "./lib/index.js",
-  "module": "./lib-esm/index.js",
-  "types": "./lib/index.d.ts",
+  "version": "4.0.2",
```

Comment:
Why is the version number changed?

Reply:
Have overseen this change during conflict resolution.